keyvault.tf

```diff
@@ -1,7 +1,106 @@
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "key-vault-rg" {
+  name     = "${var.keyVault}-rg"
+  location = var.location
+}
+
+resource "azurerm_key_vault" "simphera-key-vault" {
+  name                        = var.keyVault
```

Comment:
I suggest to remove the variable `keyVault` and use something similar as "${var.infrastructurename}-keyvault". But we need to make sure the name fulfills the naming conventions of Azure (probably the name needs to be globally unique)

Reply:
This might be helpful: https://github.com/dspace-group/simphera-reference-architecture-azure/blob/2362725d712a2bbe38e3c28b7ccd5fb4813a5f2b/modules/simphera_instance/minio-storage.tf#L16